client/src/components/MultiJurisdictionAudit/StatusBox/index.test.tsx

```diff
@@ -278,16 +278,20 @@ describe('StatusBox', () => {
           />
         </Router>
       )
-      fireEvent.click(
-        screen.getByRole('button', { name: 'Download Audit Report' }),
-        {
-          bubbles: true,
-        }
+      const downloadReportButton = screen.getByRole('button', {
+        name: 'Download Audit Report',
+      })
+      fireEvent.click(downloadReportButton, {
+        bubbles: true,
+      })
+      await expect(downloadReportButton.classList.contains('bp3-loading')).toBe(
+        true
       )
       await waitFor(() => {
         expect(window.open).toHaveBeenCalledTimes(1)
         expect(window.open).toBeCalledWith(`/api/election/1/report`)
       })
+      // haven't found a way to close the window in order to test that the spinner ceases to spin
```

Comment:
maybe just call `window.onbeforeunload` to simulate it

Reply:
@jonahkagan I had tried that a few different ways, but the `window.open` opens a new window, and I wasn't able to figure out how to trigger `onbeforeunload` for that new window.

Reply:
since here we're mocking `window.open`, maybe that mock can return a mock `Window` object representing the new window

Reply:
Tried that too, but it was returning undefined. :stuck_out_tongue_closed_eyes:

Reply:
this works for me:

```ts
    it('downloads audit report', async () => {
      const mockDownloadWindow: { onbeforeunload?: () => void } = {}
      window.open = jest.fn().mockReturnValue(mockDownloadWindow)
      render(
        <Router>
          <AuditAdminStatusBox
            rounds={roundMocks.singleComplete}
            startNextRound={jest.fn()}
            jurisdictions={jurisdictionMocks.allComplete}
            contests={contestMocks.filledTargeted.contests}
            auditSettings={auditSettings.all}
          />
        </Router>
      )
      const downloadReportButton = screen.getByRole('button', {
        name: 'Download Audit Report',
      })
      userEvent.click(downloadReportButton)
      expect(downloadReportButton).toBeDisabled()
      await waitFor(() => {
        expect(window.open).toHaveBeenCalledTimes(1)
        expect(window.open).toBeCalledWith(`/api/election/1/report`)
      })
      mockDownloadWindow.onbeforeunload!()
      await waitFor(() => {
        expect(downloadReportButton).toBeEnabled()
      })
    })
```